setup.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+maybe_install() {
+
+    exists=$(which "$1")
+    if [ "xx$exists" == "xx" ]; then
+        echo "$2 doesn't seem to be installed locally, but I can do it for you."
+        echo ""
+        echo "Hit Control-C if you don't want me to install $2 for you"
```

Comment:
Should this say "or hit enter to continue," or something similar?

Reply:
it probably should, yes.